be/src/util/mem_info.cpp

```diff
@@ -113,6 +114,16 @@ void MemInfo::process_cache_gc(int64_t& freed_mem) {
                 StoragePageCache::instance()->get_page_cache_mem_consumption(segment_v2::DATA_PAGE);
         StoragePageCache::instance()->prune(segment_v2::DATA_PAGE);
     }
+
+    if (segment_v2::InvertedIndexSearcherCache::instance()->mem_consumption() > min_free_size) {
+        freed_mem += segment_v2::InvertedIndexSearcherCache::instance()->mem_consumption();
```

Comment:
`mem_consumption` is all cache memory, but here only the expired value is released.

can let InvertedIndexSearcherCache -> prune return evict size, or do not add freed_mem here